### PR TITLE
Build/travis slack integration

### PR DESCRIPTION
**Pull request by sadovnikov**

### CODE REVIEW

File: test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java [hunk 1]
```diff
@@ -52,8 +63,8 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
 
             } else if (!currentOffer.isAcceptable()) {
                 System.out.println(
```
Comment (containersol) [inline test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)

File: test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java [hunk 2]
```diff
@@ -62,29 +73,27 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
 
     @Override
     public void offerRescinded(SchedulerDriver driver, OfferID offerId) {
+        // not supported in tests. It's ok to get task rejected
     }
 
     @Override
     public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
 
         System.out.println("Status update: task " + status.getTaskId().getValue() +
```
Comment (containersol) [inline test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)

File: test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java [hunk 3]
```diff
@@ -62,29 +73,27 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
...
-                " is in state " + status.getState().getValueDescriptor().getName());
+            " is in state " + status.getState().getValueDescriptor().getName());
 
         if (status.getState() == TaskState.TASK_LOST ||
-                status.getState() == TaskState.TASK_KILLED ||
-                status.getState() == TaskState.TASK_FAILED) {
+            status.getState() == TaskState.TASK_KILLED ||
+            status.getState() == TaskState.TASK_FAILED) {
 
             System.err.println("Aborting because task " + status.getTaskId().getValue() +
```
Comment (containersol) [inline test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)

File: test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java [hunk 4]
```diff
@@ -33,8 +44,8 @@ public void resourceOffers(SchedulerDriver driver, List<Offer> offers) {
             if (currentOffer.isAcceptable() && (acceptedOffers < MAX_OFFERS)) {
 
                 System.out.println(
```
Comment (containersol) [inline test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)

**Comment by containersol** [comment]
SonarQube analysis reported 11 issues:
* MAJOR 11 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L60) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L126) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L141) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)
* MAJOR [Move the "cpus" string literal on the left side of this string comparison.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L153) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Move the "mem" string literal on the left side of this string comparison.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L155) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Move the "ports" string literal on the left side of this string comparison.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L157) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Use isEmpty() to check whether the collection is empty or not.](https://github.com/ContainerSolutions/minimesos/blob/728ececc701fe68f477b735e932d253ab55e82d1/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L166) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1155)

**Comment by containersol** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/ContainerSolutions/minimesos/blob/60496ce82751fd3d1c9ae24b52d916fabebad074/test-framework-docker/scheduler/src/main/java/com/containersolutions/mesoshelloworld/scheduler/FrameworkScheduler.java#L146) [rule](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS106)

**Comment by frankscholten** [comment]
LGTM

**Pull request merged**